src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
For the first run through the code base it was easier to simply import everything and try and get things working. My thought was that after we got tests passing we could go back and do cleanup like this, but I think @heff has other opinions about how we handle lib / imports.